src/transaction/log_page_buffer.c

```diff
@@ -6240,6 +6277,23 @@ logpb_remove_archive_logs (THREAD_ENTRY * thread_p, const char *info_reason)
 	  min_arv_required_for_cdc--;
 	  last_deleted_arv_num = MIN (last_deleted_arv_num, min_arv_required_for_cdc);
 	}
+
```

Comment:
How about adding the and cdc case to the function comment? The comment only mentions recovery and vacuum.

Reply:
Ah, OK. I didn't consider it. Thanks.